packaging/nuget/Particular.CodeRules.nuspec

```diff
@@ -2,8 +2,10 @@
 <package xmlns="http://schemas.microsoft.com/packaging/2011/10/nuspec.xsd">
     <metadata>
         <id>Particular.CodeRules</id>
-        <version>$version$</version>
+        <version>$version$-beta</version>
```

Comment:
@danielmarbach @SimonCropp there's a special convention for installing analyzers with project.json, but for the older format (packages.config) they have to be installed via a powershell script only: https://docs.nuget.org/create/analyzers-conventions